crates/catalog/glue/tests/glue_catalog_test.rs

```diff
@@ -367,3 +368,69 @@ async fn test_list_namespace() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_update_table() -> Result<()> {
```

Comment:
should there be tests for non-happy paths?

Reply:
I think this is a good point, but I didn't test conflicts, because currently it's tricky to forge conflicts that are non-retryable without more complicated transaction actions like overwrite or rewrite